### DISCUSSION TITLE
When we will have preview6 ?

### DISCUSSION THREAD

**Discussion by glucaci** [dotnet/yarp#465] [Q&A] [answered]
When are you planning to release preview6 ? I'm interested in the Headers filter from the Route.
I saw that in master are already implemented. 👍 

Thanks!

**Comment by Tratcher** [comment, marked as answer]
Soon (?), as soon as we get https://github.com/microsoft/reverse-proxy/pull/459 merged. 

You don't need to wait though, you can try the daily build from here:
https://github.com/microsoft/reverse-proxy/blob/master/docs/DailyBuilds.md
